src/service.py

```diff
@@ -85,6 +89,8 @@ def _get(self, name):
 
     def stop(self, settings):
         print("Stopping Montagu...", flush=True)
+        if self.orderly:
+            self.orderly.kill("SIGINT")
```

Comment:
Please add a comment here mirroring the one in PR description, to explain why we are doing this.

Reply:
done